### ISSUE TITLE
0.5.5 List items invisible

### ISSUE THREAD

**Issue by 92jacko** [afollestad/material-dialogs#161]
since the 0.5.5 update, single choice lists are invisible (apart from the radio icons)

my snippet:
```
 new MaterialDialog.Builder(ctx)
                            .title("Dialog")
                            .items(new String[]{"One", "Two", "Three"})
                            .itemsCallbackSingleChoice(1, new MaterialDialog.ListCallback() {
                                @Override
                                public void onSelection(MaterialDialog dialog, View view, int which, CharSequence text) {
                                    
                                }
                            })
                            .show();
```

**Comment by TomasValenta** [comment]
In Multichoice dialog is the same situation.

**Comment by afollestad** [comment]
Fixed in release 0.5.6
